src/components/Date.vue

```diff
@@ -9,13 +9,7 @@
 		<div slot="input" class="container">
```

Comment:
I think this
```
		<div slot="input" class="container">
				<div slot="input" class="row">
```
should be
```
		<div slot="input" class="row">
			<div class="col">
```

because currently the classes go:
```
div.container (from RecordField)
  div.container (from Date)
    div.row
      div.datetimepicker.container
         ...
```

Reply:
fixed